src/js/components/Carousel/stories/InteractiveChild.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Text, Button, Box } from 'grommet';
+import { Carousel } from '../Carousel';
+
+export const InteractiveSlides = () => (
+  <Box align="center" pad="large">
+    <Carousel controls="arrows" height="medium" width="medium">
+      <Box
+        fill
+        align="center"
+        justify="center"
+        background="lavender"
+        gap="small"
+      >
+        <Text weight="bold" size="xlarge">
+          Slide 1
+        </Text>
+        <Button label="Button" />
```

Comment:
Would an onClick with an alert or console log be valuable?

Reply:
It could be, I think the fact you can clearly hover and click the button should be good. But, depending on the convention of the Grommet storybook I am open to adding whatever works best.